temoa-uiuc/data_files/11_uiuc.sql

```diff
@@ -1358,6 +1403,7 @@ INSERT INTO "CostFixed" VALUES('uiuc', 2027, 'ELCVCL', 2021, 0.00321817, 'M$/yea
 INSERT INTO "CostFixed" VALUES('uiuc', 2028, 'ELCVCL', 2021, 0.00321817, 'M$/year', 'assuming a 15 year lifespan');
 -- INSERT INTO "CostFixed" VALUES('uiuc', 2029, 'ELCVCL', 2020, 0.00321817, 'M$/year', 'assuming a 15 year lifespan');
 -- INSERT INTO "CostFixed" VALUES('uiuc', 2030, 'ELCVCL', 2020, 0.00321817, 'M$/year', 'assuming a 15 year lifespan');
+>>>>>>> upstream/master
```

Comment:
You have to pick one of these two versions to use.

Reply:
@yardasol  this needs fixing
If we merged this PR, we would be changing 11_uiuc.sql

Reply:
^^^